pkg/cortex/modules.go

```diff
@@ -157,7 +157,7 @@ func (t *Cortex) initRuntimeConfig() (services.Service, error) {
 	// make sure to set default limits before we start loading configuration into memory
 	validation.SetDefaultLimitsForYAMLUnmarshalling(t.Cfg.LimitsConfig)
 
-	serv, err := runtimeconfig.NewRuntimeConfigManager(t.Cfg.RuntimeConfig, prometheus.DefaultRegisterer)
+	serv, err := runtimeconfig.New(t.Cfg.RuntimeConfig, prometheus.DefaultRegisterer, util_log.Logger)
```

Comment:
We need to prefix metrics with "cortex_", otherwise existing Cortex metrics will be renamed.

Reply:
Oh thank you, that totally slipped my mind!